src/components/main-form.js

```diff
@@ -568,44 +557,45 @@ class MainForm extends React.Component {
               {longerOption.term} year investment total (
               <b>{formatMoney(longerOption.fv)}</b>).
             </p>
-            <Col>
+          </div>
```

Comment:
😄 As it turns out, my commit before this was an incorrect use of rows and columns. Rows and columns should always go together, and this content didn't need to be its own row column, so we can just put it on the outside